pkg/console/subresource/deployment/deployment.go

```diff
@@ -26,24 +31,18 @@ const (
 	consoleTargetPort      = 8443
 	publicURLName          = "BRIDGE_DEVELOPER_CONSOLE_URL"
 	ConsoleServingCertName = "console-serving-cert"
-	ConsoleOauthConfigName = "console-oauth-config"
+	ConsoleOauthConfigName = "console-oauth-operatorConfig"
 	ConsoleReplicas        = 2
 )
 
 const (
-	configMapResourceVersionAnnotation          = "console.openshift.io/console-config-version"
-	serviceCAConfigMapResourceVersionAnnotation = "console.openshift.io/service-ca-config-version"
-	trustedCAConfigMapResourceVersionAnnotation = "console.openshift.io/trusted-ca-config-version"
-	secretResourceVersionAnnotation             = "console.openshift.io/oauth-secret-version"
-	consoleImageAnnotation                      = "console.openshift.io/image"
+	consoleImageAnnotation   = "console.openshift.io/image"
+	deploymentVersionHashKey = "console.openshift.io/rsv"
```

Comment:
Consolidate the various resource versions down into a single hash value, so its less tedious to add more if we need.